docs/samples/v1beta1/torchserve/docs/autoscaling.md

```diff
@@ -0,0 +1,90 @@
+# Autoscaling
+
+## Deploymnet yaml
+
+For example, specify a “concurrency target” of “10”, the autoscaler will try to make sure that every replica receives on average 10 requests at a time.
+By default the pod scale with concurrency metrics
+
+Refer [model archive file generation](../model-archiver/README.md) for PV, PVC storage and marfile creation.
+
+autoscaling.yaml
+```yaml
+apiVersion: "serving.kubeflow.org/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "torchserve"
+  annotations:
+    autoscaling.knative.dev/target: "5"
+"spec:
+  predictor:
+    pytorch:
+      protocolVersion: v2
+      storageUri: "pvc://model-pv-claim"
+```
+
+## Create the InferenceService
+
+Apply the CRD
+
+```bash
+kubectl apply -f torchserve.yaml
+```
+
+Expected Output
+
+```bash
+$inferenceservice.serving.kubeflow.org/torchserve created
+```
+
+## Run a prediction
+
+The first step is to [determine the ingress IP and ports](../../../README.md#determine-the-ingress-ip-and-ports) and set `INGRESS_HOST` and `INGRESS_PORT`
+
+```bash
+MODEL_NAME=mnist
+SERVICE_HOSTNAME=$(kubectl get inferenceservice torchserve -o jsonpath='{.status.url}' | cut -d "/" -f 3)
+
+curl -v -H "Host: ${SERVICE_HOSTNAME}" http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/${MODEL_NAME}:predict -d @./mnist.json
```

Comment:
use the steps from [here](https://github.com/pytorch/serve/blob/master/model-archiver/README.md) instead of curl.